src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="
+          width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0"
+        >
+    <title>Product card</title>
+    <link rel="stylesheet" href="./main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+
+      <img
+        class="product-card__img"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+
+      <p class="product-card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```

Comment:
Try to follow this code style
```suggestion
      <p class="product-card__name">
      	Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </p>
```
Apply it to the entire code